deliveries/rb_substrate_client_milestone_1.md

```diff
@@ -0,0 +1,18 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account, which is responsible for the pull request of the accepted application is allowed to submit milestones. 
+> 
+> Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** https://github.com/uni-arts-chain/sr25519/blob/master/README.md
+* **Milestone Number:** 1
+
+| Number | Deliverable   | Link                                                                          | Notes |
+| ------ | ------------- | ----------------------------------------------------------------------------- | ----- |
+| 1.     | License       | https://github.com/uni-arts-chain/sr25519/blob/master/LICENSE.txt             |       |
+| 2.     | Documentation | https://github.com/uni-arts-chain/sr25519/blob/master/README.md#usage         |       |
+| 3.     | Testing Guide | https://github.com/uni-arts-chain/sr25519/blob/master/README.md#running-tests |       |
+| 4.     | Docker        | https://github.com/uni-arts-chain/sr25519/blob/master/Dockerfile              |       |
+
```

Comment:
This table should correspond 1-1 to the milestones laid out in the original [Application Document](https://github.com/w3f/Grants-Program/blob/master/applications/rb_substrate_client.md). It looks like one of the deliverables is still missing now: api. I assume it's part of the same repository, but ideally each of the sub-deliverables (e.g. `SR25519.keypair_from_seed(seed)`, `ED25519.verify(address, message, signature_result)` is listed with a relevant link to its location in your repository.